curriculum/challenges/english/10-coding-interview-prep/rosetta-code/day-of-the-week.english.md

```diff
@@ -23,11 +23,11 @@ Write a function that takes a start year and an end year and return an array of
 tests:
   - text: <code>findXmasSunday</code> should be a function.
     testString: assert(typeof findXmasSunday === 'function');
-  - text: <code>findChristmasSunday(2000, 2100)</code> should return an array.
+  - text: <code>findXmasSunday(2000, 2100)</code> should return an array.
     testString: assert(typeof findXmasSunday(2000, 2100) === 'object');
-  - text: <code>findChristmasSunday(2008, 2121</code> should return [1977, 1983, 1988, 1994, 2005, 2011, 2016]
+  - text: <code>findXmasSunday(2008, 2121</code> should return [1977, 1983, 1988, 1994, 2005, 2011, 2016]
```

Comment:
Just noticed, this text was missing a closing parentheses for the function call.
```suggestion
  - text: <code>findXmasSunday(2008, 2121)</code> should return [1977, 1983, 1988, 1994, 2005, 2011, 2016]
```